### ISSUE TITLE
"Configure Quality of Service for Pods" page doesn't explain how QoS classes affects scheduling

### ISSUE THREAD

**Issue by nathanperkins** [kubernetes/website#38877]
The intro for this page says

```
This page shows how to configure Pods so that they will be assigned particular Quality of Service (QoS) classes. Kubernetes uses QoS classes to make decisions about scheduling and evicting Pods.
```

There is no explanation of how Kubernetes uses QoS classes to make decisions about scheduling and evicting pods. It would be good to add a section explaining or link to a document where this is explained.

**Comment by sftim** [comment]
I don't think QoS classes do get used for that. What I learned is that they're used within the node, by the kubelet, to make placement decisions for specific resources within the node (such as topology-aware assignment to CPUs).

I could be wrong on this; there are lots of places across the Kubernetes code to look at for details.

/language en

We should revise https://kubernetes.io/docs/tasks/configure-pod-container/quality-service-pod/ and [add a QoS class concept page](https://github.com/kubernetes/website/issues/38588).

**Comment by tengqm** [comment]
I believe this issue can be closed given the clarification in #39118 and some follow-ups to that.
/close

Feel free to reopen if there are still further works to do.
